sdk/servicebus/azure-servicebus/azure/servicebus/management/aio/_management_client_async.py

```diff
@@ -0,0 +1,259 @@
+        self._config = ServiceBusManagementClientConfiguration(self._endpoint, **kwargs)
+        self._pipeline = self._build_pipeline()
+        self._impl = ServiceBusManagementClientImpl(endpoint=fully_qualified_namespace, pipeline=self._pipeline)
+
+    def _build_pipeline(self, **kwargs):  # pylint: disable=no-self-use
+        transport = kwargs.get('transport')
+        policies = kwargs.get('policies')
+        credential_policy = \
+            AsyncServiceBusSharedKeyCredentialPolicy(self._endpoint, self._credential, "Authorization") \
+            if isinstance(self._credential, ServiceBusSharedKeyCredential) \
+            else AsyncBearerTokenCredentialPolicy(self._credential, JWT_TOKEN_SCOPE)
+        if policies is None:  # [] is a valid policy list
+            policies = [
+                RequestIdPolicy(**kwargs),
+                self._config.headers_policy,
+                self._config.user_agent_policy,
+                self._config.proxy_policy,
+                ContentDecodePolicy(**kwargs),
+                self._config.redirect_policy,
+                self._config.retry_policy,
+                credential_policy,
+                self._config.logging_policy,
+                DistributedTracingPolicy(**kwargs),
+                HttpLoggingPolicy(**kwargs),
+            ]
+        if not transport:
+            transport = AioHttpTransport(**kwargs)
+        return AsyncPipeline(transport, policies)
+
+    @classmethod
+    def from_connection_string(cls, connection_string, **kwargs):
+        # type: (str, Any) -> ServiceBusManagementClient
+        """Create a client from connection string
+
+        :param str connection_string: The connection string of the Service Bus Namespace
+        """
+        endpoint, shared_access_key_name, shared_access_key, _ = parse_conn_str(connection_string)
+        if "//" in endpoint:
+            endpoint = endpoint[endpoint.index("//")+2:]
+        return cls(endpoint, ServiceBusSharedKeyCredential(shared_access_key_name, shared_access_key), **kwargs)
+
+    async def _get_queue_object(self, queue_name, clazz):
+        # type: (str, Type[Model]) -> Union[QueueDescription, QueueRuntimeInfo]
+        if not queue_name:
+            raise ValueError("queue_name must be a non-empty str")
+        with _handle_response_error():
+            et = cast(
+                ElementTree,
+                await self._impl.queue.get(queue_name, enrich=False, api_version=constants.API_VERSION)
+            )
+        return _convert_xml_to_object(queue_name, et, clazz)
+
+    async def _list_queues(self, skip, max_count, clazz):
+        # type: (int, int, Type[Model]) -> Union[List[QueueDescription], List[QueueRuntimeInfo]]
+        with _handle_response_error():
+            et = cast(
+                ElementTree,
+                await self._impl.list_entities(
+                    entity_type="queues", skip=skip, top=max_count, api_version=constants.API_VERSION
```

Comment:
do we want to contanstify this str -- "queues"

so that we would making this
```
ENTITY_TYPE_QUEUE = "queues"
ENTITY_TYPE_TOPIC = "topics
...
```

Reply:
Updated as suggested